config/daemonconfig/daemonconfig.go

```diff
@@ -136,6 +138,46 @@ func DumpConfigString(c interface{}) (string, error) {
 	return string(b), err
 }
 
+func IsExistDir(path string) bool {
+	s, err := os.Stat(path)
+	if err != nil {
+		return false
```

Comment:
It's better not to ignore errors that are not ENOEXIT

Reply:
Yes, fixed.